account_financial_report/report/open_items_xlsx.py

```diff
@@ -9,8 +9,10 @@ class OpenItemsXslx(models.AbstractModel):
     _name = 'report.a_f_r.report_open_items_xlsx'
     _inherit = 'report.account_financial_report.abstract_report_xlsx'
 
-    def _get_report_name(self):
-        return _('Open Items')
+    def _get_report_name(self, objects):
+        report = objects
+        return _('Trial Balance - %s - %s' % (
```

Comment:
Wrong string : Trial Balance -> Open Items
Other problem: It seems to me that the "_" method should be closed before string replacement i.e. _('Open Items - %Ss- %s) % (report.company_id.name, ...). If you agree on this second point, it should be changed in the _get_report_name() methods of all the _xlsx.py files.

Reply:
@jbeficent Do you agree with my 2 remarks ?

Reply:
@alexis-via I have applied the fixes according to your remarks. Once Travis is Green can you give your approval? Once this is merged I will backport to 10.0